### DISCUSSION TITLE
Is there a command available to query fusion workflows settings and execution logs?

### DISCUSSION THREAD

**Discussion by plus-zz** [CrowdStrike/psfalcon#386] [Q&A]
Hi @bk-cs,

do you have something to query the fusion workflows?
I'm looking for an export capability about the trigger and actions to check for similarities to consolidate workflows.
Something like get-FalconFusionWorkflows?

Second mission, checking for not needed workflows anymore. How often do they trigger?
Is there something like, get-FalconFusionExecutionLog?

Thank you for your support.

**Comment by bk-cs** [comment]
These APIs were recently introduced and have not been added to PSFalcon yet. I expect that they will be available in the next release.
